rero_ils/modules/items/api/record.py

```diff
@@ -435,6 +435,21 @@ def issue_inherited_first_call_number(self):
             return Holding.get_record_by_pid(
                 self.holding_pid).get('call_number')
 
+    @property
+    def provisional_inherited_call_numbers(self):
+        """Get provisional inherited first and second call number.
+
+        For item of type provisional,
+        returns the parent holdings first and second call numbers if exist.
+        """
+        from ...holdings.api import Holding
+        if self.get('type') == 'provisional' and not self.get('call_number'):
```

Comment:
There is a case when the `provisional` item has a `second_call_number`, in this case you will not display the item second call number at all.